app/services/get-local-mods.ts

```diff
@@ -30,81 +30,214 @@ function getOwml(owmlPath: string) {
   return owml;
 }
 
-export function getLocalMods(owmlPath: string) {
-  if (!owmlPath) {
+function getBepInEx(alphaPath: string) {
+  const bepInExManifestPath = `${alphaPath}/BepInEx.Manifest.json`;
+  const bepInExManifest: Manifest = fs.existsSync(bepInExManifestPath)
+    ? fs.readJSONSync(bepInExManifestPath)
+    : null;
+  const bepInEx: Mod = {
+    name: bepInExManifest?.name ?? 'BepInEx',
+    author: bepInExManifest?.author ?? 'bbepis',
+    uniqueName: bepInExManifest?.uniqueName ?? 'bbepis.BepInEx',
+    modPath: alphaPath,
+    localVersion: bepInExManifest
+      ? bepInExManifest?.version ?? 'v5.4.19'
+      : undefined,
+    isEnabled: true,
+    isRequired: true,
+    errors: [],
+    dependencies: [],
+    addons: [],
+    isAlpha: true,
+  };
+  return bepInEx;
+}
+
+function getCmowa(cmowaPath: string) {
+  const cmowaManifestPath = `${cmowaPath}/CMOWA.Manifest.json`;
+  const cmowaManifest: Manifest = fs.existsSync(cmowaManifestPath)
+    ? fs.readJSONSync(cmowaManifestPath)
+    : null;
+  const cmowa: Mod = {
+    name: cmowaManifest?.name ?? 'CMOWA',
+    author: cmowaManifest?.author ?? 'Locochoco',
+    uniqueName: cmowaManifest?.uniqueName ?? 'Locochoco.CMOWA',
+    modPath: cmowaPath,
+    localVersion: cmowaManifest
+      ? cmowaManifest?.version ?? 'v1.0.1'
+      : undefined,
+    isEnabled: true,
+    isRequired: true,
+    errors: [],
+    dependencies: ['bbepis.BepInEx'],
+    addons: [],
+    isAlpha: true,
+  };
+  return cmowa;
+}
```

Comment:
The reason OWML had some of this strange hardcoded stuff is because early OWML versions had no manifest.json. If these required items always have their own manifests, then the hardcoded strings are probably not needed. I'm just making sure you didn't just copy this part without thinking because you saw it being done for OWML. Don't know if it's actually needed or not. I leave it up to you.

Reply:
It is needed for when the user doesn't have the CMOWA or BepInEx installed and runs the mod manager without internet.
If they do that without the hardcoded strings, the 2 mods don't show up in the mods tab.